navigation/src/main/java/com/graphhopper/navigation/NavigateResponseConverter.java

```diff
@@ -260,6 +309,37 @@ private static ObjectNode putInstruction(PointList points, InstructionList instr
         Instruction instruction = instructions.get(instructionIndex);
         ArrayNode intersections = instructionJson.putArray("intersections");
 
+        // make pointList writeable
+        PointList pointList = instruction.getPoints().clone(false);
+
+        if (instructionIndex == 0) {
+             // Fix first (departure) instruction
+             // to only have one "out" intersection
+            intersectionDetails = fixFirstIntersectionDetail(intersectionDetails);
+        }  else if (instructionIndex < instructions.size() - 1) {
+            // modify pointlist to include the first point of the next instruction
+            // for all instructions but the arrival
+            PointList nextPoints = instructions.get(instructionIndex + 1).getPoints();
```

Comment:
We ran this code also for the first instruction before, now this isn't run for the first instruction anymore. Is this desired, and if yes, why?